extensions/amp-story/1.0/amp-story.js

```diff
@@ -805,6 +812,47 @@ export class AmpStory extends AMP.BaseElement {
   }
 
 
+  /**
+   * @private
+   */
+  initializeStoryAccess_() {
+    Services.accessServiceForDocOrNull(this.getAmpDoc()).then(accessService => {
```

Comment:
Is it possible that there's a race condition and the access service hasn't been registered yet, even though there will eventually be one?

Reply:
I'm not an expert about the AMP services, but I think the Promise based implementation is meant to protect us from that.
We had issues with our `storeService` internally because we were using the synchronous version

Reply:
Whoops, I think that's right.